adapters/cpmstar/cpmstar.go

```diff
@@ -0,0 +1,158 @@
+package cpmstar
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+//Adapter :
```

Comment:
I'm guessing you are adding these comments to avoid linter warnings. Please consider providing a simple comment rather than leaving it empty. If you want to omit these kinds of comments, that's fine by me.